src/cwe_checker_lib/src/analysis/forward_interprocedural_fixpoint.rs

```diff
@@ -283,3 +284,149 @@ pub fn create_computation_with_alternate_worklist_order<'a, T: Context<'a>>(
         default_value.map(NodeValue::Value),
     )
 }
+
+/// Generate a new computation from the corresponding context and an optional default value for nodes.
+/// Uses the Callee before Caller worklist order when computing the fixpoint.
+pub fn create_computation_with_callee_before_caller_worklist_order<'a, T: Context<'a>>(
+    problem: T,
+    default_value: Option<T::Value>,
+) -> super::fixpoint::Computation<GeneralizedContext<'a, T>> {
+    let mut graph = problem.get_graph().clone();
+    graph.retain_edges(|frozen, edge| !matches!(frozen[edge], Edge::Call(..)));
+    let priority_sorted_nodes: Vec<NodeIndex> = petgraph::algo::kosaraju_scc(&graph)
+        .into_iter()
+        .rev()
```

Comment:
This `rev()` confuses me. We did not reverse the order in the old node priority lists. Was this wrong?

Edit: Oh, maybe I explained the order logic the wrong way around. Nodes with a higher index in the priority list (i.e. towards the end of the list) have a higher priority in the worklist algorithm (i.e. get stabilized sooner). So I think this `rev()` is wrong here.